src/components/buttons/flat-button/flat-button.js

```diff
@@ -37,9 +38,10 @@ const getTextColor = (tone, isHover = false) => {
 
 export const FlatButton = props => {
   const dataProps = {
-    'data-track-component': 'LinkButton',
+    'data-track-component': 'FlatButton',
     ...filterAriaAttributes(props),
     ...filterDataAttributes(props),
+    ...filterEventAttributes(props),
```

Comment:
Is this function exported from UI-Kit then for app level components to spread event props? I still wonder if we should rather export a

```js
import { Tooltip } from 'ui-kit';

const Component = Tooltip.forwardProps(props => <div {...props} /> 
```

which would internally can use the `filterEeventAttributes` but I like the api more as the origin of the props is clear and the Tooltip gets the change to potentially spead any other props or even refs (2nd arg).

Reply:
I don't really see the point of having this functionality when most people will just follow the readme and do something like 

js```
const Component = props => (
<div {...props}>Foo</div>
)```

This forwarding props only making sense because we are so against prop forwarding, but that's a technical decision we don't need to force on others...

Reply:
Ok.